docs/UsersGuide/source/BackgroundInfo/Introduction.rst

```diff
@@ -60,7 +60,7 @@ Building, Running, and Testing the SRW App
 .. hint:: 
    * To get started with the SRW App, it is recommended that users try one of the following options: 
 
-      #. View :numref:`Section %s: Quick Start Guide <NCQuickstart>` for a quick overview of the workflow steps. Especially helpful for users with access to a `Level 1 platform <https://github.com/ufs-community/ufs-srweather-app/wiki/Supported-Platforms-and-Compilers>`__.
+      #. View :numref:`Section %s: Quick Start Guide <NCQuickstart>` for a quick overview of the workflow steps. Especially helpful for users with access to a :srw-wiki:`Level 1 platform <Supported-Platforms-and-Compilers>`.
```

Comment:
Suggested: Level 1 platforms

Reply:
I think it's probably fine as-is because a user might have access to one platform or more than one. Grammatically, "access to a Level 1 platform" or "access to Level 1 platforms" are both correct. :)